src/create.wsgi

```diff
@@ -164,7 +164,7 @@ def application(environ, start_response):
 
         if unpack_retcode != 0:
             raise Exception
-    except:
+    except OSError:
```

Comment:
Is this right? The line just above raises an `Exception` exception.

Reply:
:man_facepalming: